runtime/queries/less/highlights.scm

```diff
@@ -0,0 +1,35 @@
+; inherits: css
+
+[
+  "@import"
+  "@namespace"
+  "@charset"
+] @keyword
+
+(js_comment) @comment @spell
```

Comment:
```suggestion
(js_comment) @comment
```

Neovim tree-sitter queries sometimes have `@spell` captures that they use to tag a range to be spell checked. We don't have spellchecking support yet but when we do it will be a separate query file, so we can remove this capture here